### DISCUSSION TITLE
Why package manager ask me about GitHub username and pass even if password auth is not supported

### DISCUSSION THREAD

**Discussion by mswiecic** [sxyazi/yazi#1535] [1. Q&A] [answered]
Why package manager ask me about GitHub username and pass even if password authentication is not supported. I have configured ssh access to GitHub and this channel should be used.
![obraz](https://github.com/user-attachments/assets/a6568cfc-1eb5-437c-ba8f-f1c8df86a793)

**Comment by sxyazi** [comment, marked as answer]
There is no repository called https://github.com/yazi-rs/plugins.max-preview.yazi. When you try to clone a repository that is inaccessible or doesn't exist, `git` will prompt you to enter a password to confirm whether it's a permissions issue.

**Reply by mswiecic** [reply]
:man_facepalming: a colon in the name of repo was the reason.
